packages/legacy-client/src/index.ts

```diff
@@ -141,6 +170,7 @@ export type {
   InvalidSortOrder,
   InvalidSortType,
   InvalidUserId,
+  LinearRing,
```

Comment:
This entire set of additional APIs feels really bad that we're exporting them, especially without renames.